### PR TITLE
mysql: add customized release version config option

### PR DESCRIPTION
**Pull request by AilinKid**
Signed-off-by: ailinkid <314806019@qq.com>

### What problem does this PR solve? 
Add customized release version config option which will be initialized in the TiDB makefile.

### What is changed and how it works?
Add customized release version config option which will be initialized in the TiDB makefile.
Related-TiDB-PR https://github.com/pingcap/tidb/pull/27692

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Comment by tangenta** [comment]
Can we move this constant to TiDB?

File: mysql/const.go [hunk 1]
```diff
@@ -30,10 +30,19 @@ var (
 	// TiDBReleaseVersion is initialized by (git describe --tags) in Makefile.
 	TiDBReleaseVersion = "None"
 
+	// CustomizedReleaseVersion is initialized by (your definition) in Makefile.
+	CustomizedReleaseVersion = "None"
+
 	// ServerVersion is the version information of this tidb-server in MySQL's format.
-	ServerVersion = fmt.Sprintf("5.7.25-TiDB-%s", TiDBReleaseVersion)
+	ServerVersion = fmt.Sprintf("5.7.25-%s", CustomizedReleaseVersion)
```
Comment (zimulala) [inline mysql/const.go:37]: Why not use `TiDBReleaseVersion` as the default value?

Comment (AilinKid) [reply mysql/const.go:37]: refer the related TiDB pr to see what user want
```
select version()
UbiSQL-v1.0.0-rc1
```

**Review by tangenta — APPROVED** [review]

File: mysql/const.go (hunk 1 above)
Comment (zimulala) [reply mysql/const.go:37]: I mean, the default is the TiDB release version. After the user sets it, we change the value.

**Review by zimulala — APPROVED** [review]
LGTM

**Pull request merged**